src/app/backend/resource/deployment/deploy.go

```diff
@@ -285,55 +290,64 @@ func getLabelsMap(labels []Label) map[string]string {
 	return result
 }
 
-type createObjectFromInfo func(info *kubectlResource.Info) (bool, error)
-
-// CreateObjectFromInfoFn is an implementation of createObjectFromInfo
-func CreateObjectFromInfoFn(info *kubectlResource.Info) (bool, error) {
-	createdResource, err := kubectlResource.NewHelper(info.Client, info.Mapping).Create(info.Namespace, true, info.Object)
-	return createdResource != nil, err
-}
-
 // DeployAppFromFile deploys an app based on the given yaml or json file.
-func DeployAppFromFile(cfg clientcmd.ClientConfig, spec *AppDeploymentFromFileSpec,
-	createObjectFromInfoFn createObjectFromInfo) (bool, error) {
-	const emptyCacheDir = ""
-	validate := spec.Validate
+func DeployAppFromFile(cfg *rest.Config, spec *AppDeploymentFromFileSpec) (bool, error) {
+	reader := strings.NewReader(spec.Content)
+	log.Printf("Namespace for deploy from file: %s\n", spec.Namespace)
+	d := yaml.NewYAMLOrJSONDecoder(reader, 4096)
+	for {
+		data := unstructured.Unstructured{}
+		if err := d.Decode(&data); err != nil {
+			if err == io.EOF {
+				return true, nil
```

Comment:
because data may be a list ,so it need read by  for cycle,when it read at the end of file, it will throw io.EOF, That's just what I expected and I need return true .this idea Refer to kubernetes cli